### PR TITLE
[new release] cf and cf-lwt (0.5.0)

### PR DESCRIPTION
**Pull request by samoht**
OCaml bindings to macOS CoreFoundation

- Project page: <a href="https://github.com/mirage/ocaml-cf">https://github.com/mirage/ocaml-cf</a>
- Documentation: <a href="https://mirage.github.io/ocaml-cf/">https://mirage.github.io/ocaml-cf/</a>

##### CHANGES:

* Fix build with `opam-monorepo` (mirage/ocaml-cf#2, @samoht)
* Update ocamlformat config to 0.21.0 (mirage/ocaml-cf#3. @samoht)

### CODE REVIEW

File: packages/cf/cf.0.5.0/opam [hunk 1]
```diff
@@ -10,7 +10,7 @@ homepage: "https://github.com/mirage/ocaml-cf"
 doc: "https://mirage.github.io/ocaml-cf/"
 bug-reports: "https://github.com/mirage/ocaml-cf/issues"
 depends: [
-  "dune" {>= "2.9"}
+  "dune" {>= "3.0"}
```
Comment (samoht) [inline packages/cf/cf.0.5.0/opam:13]: do you remember what was the error message?

**Comment by samoht** [comment]
Trying to fix the `irmin-unix` tests here: https://github.com/ocaml/opam-repository/pull/21117

File: packages/cf/cf.0.5.0/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/cf/cf.0.5.0/opam:13]: yes, see my previous comment above: https://github.com/ocaml/opam-repository/pull/21090#discussion_r841961613

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by kit-ty-kate** [comment]
Could you return the fixes to the opam file upstream?

**Comment by samoht** [comment]
Done in https://github.com/mirage/ocaml-cf/pull/7

**Pull request merged**